builder/parser/line_parsers.go

```diff
@@ -149,7 +149,7 @@ func parseEnv(rest string) (*Node, map[string]bool, error) {
 		strs := TOKEN_WHITESPACE.Split(rest, 2)
 
 		if len(strs) < 2 {
-			return nil, nil, fmt.Errorf("ENV must have two arguments")
+			return nil, nil, fmt.Errorf(key + " must have two arguments")
```

Comment:
I understand that this it heritage from `ENV`, but isn't it weird that command-line allows for valueless labels (e.g., `--label plop`) while Dockerfile instruction doesn't (e.g., `LABEL plop` yield `LABEL must have two arguments`)?